__tests__/integration.test.js

```diff
@@ -254,5 +254,17 @@ describe('/api/users', () => {
        })
     });  
  })
-
+ describe('/api/articles', () => {
+    test('GET: 200, returns articles of a specified topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+        const { articles } = body
+        articles.forEach((article) => {
+            expect(article.topic).toBe('mitch')
+        })
+        })
```

Comment:
This is very clean, I would also include a check for the length of the array to ensure it isn't empty as well